nameable.rb

```diff
@@ -0,0 +1 @@
+
```

Comment:
- [ ] Namable class should have a method 'correct_name' rasing an error. It should look something like the following:
![image](https://user-images.githubusercontent.com/81932478/215619342-b4261369-4418-4ca2-8f3e-bd94386458b4.png)
- Kindly fix this as it is a requirement

Reply:
well resolved! ✔️